ansible/roles_ocp_workloads/ocp4_workload_dso/tasks/nexus.yml

```diff
@@ -26,7 +31,9 @@
     - project_admin|trim() != ""
 
 - name: annotate project project {{ ocp4_dso_nexus_project_name }}
-  shell: "{{ ocp4_dso_openshift_cli }} annotate --overwrite namespace {{ ocp4_dso_nexus_project_name }} {{ project_annotations }}"
+  shell: >-
```

Comment:
So long as doing revisions... please change `shell` to `command` here as it is not using any shell functions.